storage/storage.go

```diff
@@ -47,31 +48,71 @@ func NewClient(url *objurl.ObjectURL) (Storage, error) {
 
 // Object is a generic type which contains metadata for storage items.
 type Object struct {
-	URL          *objurl.ObjectURL
-	Etag         string
-	ModTime      time.Time
-	Mode         os.FileMode
-	Size         int64
-	StorageClass StorageClass
-	Err          error
+	URL          *objurl.ObjectURL `json:"key,omitempty"`
+	Etag         string            `json:"etag,omitempty"`
+	ModTime      *time.Time        `json:"last_modified,omitempty"`
```

Comment:
Why is this change necessary?

Reply:
In JSON output, non-pointer time.Time prints default time even if it is not given. That is why I updated it to pointer and put `omitempty` into its struct tag.